index.html

```diff
@@ -289,7 +289,114 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+       <div id="contact">
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
```

Comment:
According to our Figma desing the text is aligned to the left, but it's aligned to the right on your project. So let's follow the Figma design, and fix this.

![image](https://user-images.githubusercontent.com/34779928/190403053-44b9e986-c2b8-42b2-9389-7936db5123a0.png)